1-introduction/1.3.1-unix.md

```diff
@@ -1,4 +1,5 @@
 # Unix Basics
+Unix us used as a generic term to refer to Unix like operating systems (i.e. Linux, BSD, MacOS). Windows is not a Unix operating system, but has Windows Subsystem for Linux (WSL) to provide Unix functionality on Windows computers.
```

Comment:
- I think it would be better to say Unix based systems. as they listed systems are not "like" Unix they are directly based on it.
- as far as I know WSL provides non-posix compliant emulation so Im not sure its worth listing here as Unix functional.

Reply:
I think wsl2 is POSIX compliant now given it actually runs the full Linux Kernel. As this doc shows this was not the case in the original implementation of wsl. But wsl2 has been the default for a while now and I don't see them going back. https://learn.microsoft.com/en-us/windows/wsl/compare-versions